pkg/yurttunnel/constants/constants.go

```diff
@@ -37,6 +38,7 @@ const (
 	YurttunnelAgentCertDir           = "/var/lib/yurttunnel-agent/pki"
 	YurttunnelCSRApproverThreadiness = 2
 
-	YurtEdgeNodeLabel       = "alibabacloud.com/is-edge-worker"
-	YurttunnelAgentPodIPEnv = "POD_IP"
+	YurtEdgeNodeLabel          = "alibabacloud.com/is-edge-worker"
+	YurttunnelEnableAgentLabel = "alibabacloud.com/enable-yurttunnel-agent"
```

Comment:
the label for tunnel agent  should keep consistency with old version

Reply:
initialize the value based on the user input